LICENSES

```diff
@@ -11,7 +11,7 @@ github.com/Azure/go-ntlmssp,https://github.com/Azure/go-ntlmssp/blob/754e6932135
 github.com/NYTimes/gziphandler,https://github.com/NYTimes/gziphandler/blob/v1.1.1/LICENSE,Apache-2.0
 github.com/Venafi/vcert/v5,https://github.com/Venafi/vcert/blob/v5.3.0/LICENSE,Apache-2.0
 github.com/akamai/AkamaiOPEN-edgegrid-golang,https://github.com/akamai/AkamaiOPEN-edgegrid-golang/blob/v1.2.2/LICENSE,Apache-2.0
-github.com/antlr/antlr4/runtime/Go/antlr/v4,https://github.com/antlr/antlr4/blob/8188dc5388df/runtime/Go/antlr/v4/LICENSE,BSD-3-Clause
```

Comment:
`make update-licenses` is non-deterministic.

It seems to add or remove the version segments of the license URLs each time I run it.

Reply:
I tried it and found that go-licenses consistently adds the version path (e.g., `/v2`). I ran it 10 times and it always gives the same result on my machine... that's odd

Here is the version I have: v1.6.1-0.20230831152858-9a41918e8c1e

```bash
$ go version -m _bin/tools/go-licenses
_bin/tools/go-licenses: go1.21.5
	path	github.com/google/go-licenses
	mod	github.com/google/go-licenses	v1.6.1-0.20230831152858-9a41918e8c1e	h1:U0LWvxDkDrUA2a0GPSFu94iBnRYRpA2a+vhC+ihpxpE=
```

Is that the version you are using, or maybe it is an old version of go-licenses? In december, the version of go-licenses was changed from v1.6.0 to go-licenses=9a41918e8c1e254f6472bdd8454b6030d445b255.

Reply:
I now get consistent results. Perhaps the problem was caused by the recent GitHub flakiness.
 * https://www.githubstatus.com/incidents/pxg3dz4yg7lp